tests/saw/spec/handshake/rfc_handshake_tls13.cry

```diff
@@ -305,3 +305,62 @@ rfc2S2N msg = mkAct recordType messageType writer
                  | msg.sender == both then 'B'
                  else '!'
         noMessageType = TLS_HELLO_REQUEST // HelloRequests don't exist in tls1.3
+
+/**
+ * printHandshake and s2nToWords are two helper functions for debugging cryptol
+ * Take the counterexample from tls13rfcSimulatesS2N and plug the values into
+ * testParameters and testConnection. Run printHandshake with ascii set to on. 
```

Comment:
You might want to list the exact commands to both turn on ascii and run the function with the right arguments.